packages/beacon-state-transition/src/cache/epochContext.ts

```diff
@@ -150,6 +163,9 @@ export class EpochContext {
   epoch: Epoch;
   syncPeriod: SyncPeriod;
 
+  currentProposerSeed: Uint8Array;
+  nextProposerSeed: Uint8Array;
```

Comment:
Same here, what's the advantage of keeping this around? Why not compute on demand and discard after caching `nextEpochProposers`?

Reply:
Then please add comments :pray: all non-obvious decisions must be rationalized in committed comments